drv/fpga-devices/src/ecp5.rs

```diff
@@ -0,0 +1,453 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use super::Fpga;
```

Comment:
I normally only see `use super` for in-line modules that want to use code from their parent context; it's more common to `use crate` to include things from the crate root.

```suggestion
use crate::Fpga
```